codespell_lib/data/dictionary_code.txt

```diff
@@ -24,6 +25,7 @@ nto->not
 objext->object
 od->of
 process'->process's
+protecten->protection
 referer->referrer
 rela->real
 secant->second
```

Comment:
secant:
https://dictionary.cambridge.org/de/worterbuch/englisch/secant

Reply:
Indeed `secant` is real. Maybe move to `rare`?

Reply:
Rare is used in default. This would annoy much more people than code.

Reply:
Let's just delete it for now then. It's not even that close a misspelling...